### PR TITLE
docs: OpenAPI nutrients enhancements

### PR DESCRIPTION
**Pull request by alexgarel**

### CODE REVIEW

**Review by benbenben2 — APPROVED** [review]
Looks good to me

File: docs/api/ref/schemas/product_nutrition.yaml [hunk 1]
```diff
@@ -82,7 +106,7 @@ properties:
           imports.
         example: 12.5
     patternProperties:
-      '(?<nutrient>\w+)_unit':
+      '(?<nutrient>[\w-]+)_unit':
```
Comment (stephanegigandet) [inline docs/api/ref/schemas/product_nutrition.yaml:109]: Good :)

File: docs/api/ref/schemas/product_nutrition.yaml [hunk 2]
```diff
@@ -11,15 +11,38 @@ properties:
...
+  nutrition_data_per:
+    type: string
+    enum:
+    - serving
+    - 100g
+    description: |
+      The nutrition data on the package can be per serving or per 100g.
+
+      This is essential to understand if `<nutrient>_value` and `<nutrient>` 
+      values in `nutriments` applies for a serving or for 100g.
```
Comment (stephanegigandet) [inline docs/api/ref/schemas/product_nutrition.yaml:23]: Can we add something like: "When writing products, this setting applies to all existing nutrients values for the product, not only the nutrient values sent in the write request. So it should not be changed unless all nutrients values are provided with values that match the nutrition_data_per field."

**Review by stephanegigandet — APPROVED** [review]
Thank you! I added one suggestion.

**Pull request merged**